### DISCUSSION TITLE
Please consider to allow .PT domain

### DISCUSSION THREAD

**Discussion by josesimoes** [forwardemail/forwardemail.net#304] [Q&A] [answered]
Hello,

PT is the ccTLD for Portugal. I was surprised it wasn't in the list of the supported domains.
Can you please consider adding it?

Thanks,

**Comment by titanism** [comment, marked as answer]
It is a supported domain, but only on paid plans right now.  We'll look at adding support for it on free plans.

**Reply by josesimoes** [reply]
Perfect! Thanks.

**Comment by titanism** [comment]
Portugal `.pt` support has been added to the free plan

<https://github.com/forwardemail/forwardemail.net/commit/fbdab8475b335a835ce653cdedb504f95d7b1ef3>

**Reply by josesimoes** [reply]
Awesome!
